src/Options/Applicative/Builder.hs

```diff
@@ -582,6 +583,9 @@ helpShowGlobals = PrefsMod $ \p -> p { prefHelpShowGlobal = True }
 helpIndent :: Int -> PrefsMod
 helpIndent w = PrefsMod $ \p -> p { prefTabulateFill = w }
 
+-- | Set the pretty printer for brief help text.
+briefPrettyPrinter :: BriefPrettyPrinter -> PrefsMod
+briefPrettyPrinter bpp = PrefsMod $ \p -> p { prefBriefPP = bpp }
```

Comment:
I think it would be better to do this differently, requiring the consumer of optparse to use the constructor and having it exported in types makes it a part of the "public API" which is a bit fragile, and not really in line with the other mods we have.

In my opinion it's better to instead have an option like:
```haskell
briefHangPoint :: Int -> PrefsMod
```

which sets the location where the hang constructor kicks in.


Now, when I mentioned that I thought the constructor was redundant, I was going off the definition of `hangAtIfOver` being

```haskell
hangAtIfOver :: Int -> Int -> Doc -> Doc
hangAtIfOver i j d =
  column $ \k ->
    if k <= j then
      align d
    else
      linebreak <> ifAtRoot (indent i) d
```
so if one sets this to maxBound it is functionally identical to `align`.

Reply:
Hey @HuwCampbell, I did the change and indeed it's simpler :+1:

And I confirm that setting `j` to `maxBound` is indeed functionally equivalent to the previous design :+1: